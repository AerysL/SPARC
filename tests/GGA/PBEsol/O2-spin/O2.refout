***************************************************************************
*                           SPARC (May 11, 2020)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun May 17 23:26:31 2020                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 21 18 18 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 105 90 90
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 2
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001000000336
CHEB_DEGREE: 35
NSTATES: 10
#WARNING: Number of bands may be insufficient for efficient SCF convergence.
EXCHANGE_CORRELATION: GGA_PBEsol
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 4.00E-05
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: PBEsol/O2-spin/O2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 2
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 10
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 4 2 3
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  24
Mesh spacing                       :  0.2 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  PBEsol/O2-spin/O2.out_1
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  O 6
Pseudopotential                    :  PBEsol/O2-spin/O.psp8
Pseudocharge radii of atom type 1  :  7.00 7.00 7.00 (x, y, z dir)
Number of atoms of type 1          :  2
Estimated total memory usage       :  1.09 GB
Estimated memory per processor     :  46.50 MB
========================================================================================
                    Self Consistent Field (SCF#1)                     
========================================================================================
Iteration     Free Energy (Ha/atom)    Magnetization     SCF Error        Timing (sec)
1            -1.5794355189E+01         2.0000E+00        1.468E-01        4.144
2            -1.5919251009E+01         2.0000E+00        3.044E-01        2.089
3            -1.5854480273E+01         2.0000E+00        1.117E-01        1.744
4            -1.5853774975E+01         2.0000E+00        8.844E-02        1.736
5            -1.5856401879E+01         2.0000E+00        9.801E-02        1.743
6            -1.5853427179E+01         2.0000E+00        2.940E-02        1.688
7            -1.5853398818E+01         2.0000E+00        2.805E-02        1.662
8            -1.5853113202E+01         2.0000E+00        2.704E-03        1.791
9            -1.5853119519E+01         2.0000E+00        3.818E-03        1.636
10           -1.5853117031E+01         2.0000E+00        1.295E-03        1.498
11           -1.5853116718E+01         2.0000E+00        4.022E-04        1.439
12           -1.5853116850E+01         2.0000E+00        2.356E-04        1.831
13           -1.5853116976E+01         2.0000E+00        7.895E-05        1.379
14           -1.5853116981E+01         2.0000E+00        5.398E-05        1.242
15           -1.5853116981E+01         2.0000E+00        3.159E-05        1.323
16           -1.5853116990E+01         2.0000E+00        1.024E-05        1.322
17           -1.5853116986E+01         2.0000E+00        8.178E-06        1.159
18           -1.5853117003E+01         2.0000E+00        8.475E-07        0.833
Total number of SCF: 18    

k = [0.000, 0.000, 0.000]
Occupation of state 9 = 0.000000000000000.
Occupation of state 10 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -1.5853116995E+01 (Ha/atom)
Total free energy                  : -3.1706233991E+01 (Ha)
Band structure energy              : -6.5888332164E+00 (Ha)
Exchange correlation energy        : -6.5091072314E+00 (Ha)
Self and correction energy         : -6.4063409725E+01 (Ha)
Entropy*kb*T                       : -3.4771795336E-13 (Ha)
Fermi level                        : -2.6225107222E-01 (Ha)
RMS force                          :  1.7006931771E-01 (Ha/Bohr)
Maximum force                      :  1.7006931771E-01 (Ha/Bohr)
Time for force calculation         :  0.030 (sec)
Pressure                           : -7.4116835495E-01 (GPa)
Maximum stress                     :  2.2120127515E+00 (GPa)
Time for stress calculation        :  0.063 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  31.555 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
