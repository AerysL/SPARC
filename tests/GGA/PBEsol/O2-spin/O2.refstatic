***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of O:
      0.4285714286       0.5000000000       0.5000000000
      0.5714285714       0.5000000000       0.5000000000
Total free energy (Ha): -3.170623399093295E+01
Atomic forces (Ha/Bohr):
  1.7006931771E-01  -1.4298827781E-07  -2.0658224651E-07
 -1.7006931771E-01   1.4298827781E-07   2.0658224651E-07
Stress (GPa): 
  2.2120127515E+00  -5.7684085353E-06  -6.4701122850E-06 
 -5.7684085353E-06   5.7461910386E-03  -5.2979794639E-07 
 -6.4701122850E-06  -5.2979794639E-07   5.7461223201E-03
