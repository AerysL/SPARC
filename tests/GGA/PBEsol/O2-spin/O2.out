***************************************************************************
*                           SPARC (Nov 06, 2019)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                 Start time: Thu Dec 19 14:53:53 2019                    *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 21 18 18 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 105 90 90
FD_ORDER: 12
BOUNDARY_CONDITION: 2
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 2
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001000000001
CHEB_DEGREE: 35
NSTATES: 10
EXCHANGE_CORRELATION: GGA_PBEsol
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: none
#WARNING: the preconditioner for SCF has been turned off, this 
#might lead to slow SCF convergence. To specify SCF preconditioner, 
#use 'MIXING_PRECOND' in the .inpt file
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: O2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 2
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 10
NP_DOMAIN_PARAL: 2 1 1
NP_DOMAIN_PHI_PARAL: 5 2 4
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  40
Mesh spacing                       :  0.2 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  O2.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  O 6
Pseudopotential                    :  ./O.psp8
Pseudocharge radii of atom type 1  :  7.20 7.20 7.20 (x, y, z dir)
Number of atoms of type 1          :  2
Fractional coordinates of atoms of type 1     :
      0.4285714286       0.5000000000       0.5000000000
      0.5714285714       0.5000000000       0.5000000000
========================================================================================
                    Self Consistent Field (SCF#1)                     
========================================================================================
Iteration     Free Energy (Ha/atom)    Magnetization     SCF Error        Timing (sec)
1            -1.5821716971E+01         2.0000E+00        1.240E-01        9.960
2            -1.5861788880E+01         2.0000E+00        1.727E-01        3.935
3            -1.5861112549E+01         2.0000E+00        1.617E-01        3.865
4            -1.5852479154E+01         2.0000E+00        5.448E-02        3.795
5            -1.5853216469E+01         2.0000E+00        1.733E-02        3.585
6            -1.5853118294E+01         2.0000E+00        7.237E-03        12.648
7            -1.5853117131E+01         2.0000E+00        1.577E-03        3.327
8            -1.5853116788E+01         2.0000E+00        7.651E-04        3.262
9            -1.5853116916E+01         2.0000E+00        2.821E-04        3.109
10           -1.5853116968E+01         2.0000E+00        1.094E-04        2.963
11           -1.5853116983E+01         2.0000E+00        3.504E-05        2.788
12           -1.5853116977E+01         2.0000E+00        3.507E-05        2.767
13           -1.5853116992E+01         2.0000E+00        1.285E-05        2.652
14           -1.5853116992E+01         2.0000E+00        3.858E-06        2.385
15           -1.5853117011E+01         2.0000E+00        2.849E-06        2.398
16           -1.5853117011E+01         2.0000E+00        1.175E-06        2.311
17           -1.5853116979E+01         2.0000E+00        2.792E-07        2.086
Total number of SCF: 17    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -1.5853117026E+01 (Ha/atom)
Total free energy                  : -3.1706234052E+01 (Ha)
Band structure energy              : -6.5888329055E+00 (Ha)
Exchange correlation energy        : -6.5091067015E+00 (Ha)
Self and correction energy         : -6.4063409725E+01 (Ha)
Entropy*kb*T                       : -3.5943564131E-13 (Ha)
Fermi level                        : -2.6221658550E-01 (Ha)
Atomic forces (Ha/Bohr):
  1.7006623277E-01  -1.5249877095E-07   5.1361175999E-08
 -1.7006623277E-01   1.5249877095E-07  -5.1361175999E-08
RMS force                          :  1.7006623277E-01 (Ha/Bohr)
Maximum force                      :  1.7006623277E-01 (Ha/Bohr)
Time for force calculation         :  0.108 (sec)
Stress (GPa) : 
  2.2120128029E+00  -2.0802583965E-06   1.2539353094E-06 
 -2.0802583965E-06   5.7493756273E-03   3.5613671292E-07 
  1.2539353094E-06   3.5613671292E-07   5.7491076813E-03 
Pressure                           : -7.4117042874E-01 (GPa)
Maximum stress                     :  2.2120128029E+00 (GPa)
Time for stress calculation        :  0.202 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  70.815 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
